src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb

```diff
@@ -145,13 +145,22 @@ def wait_for_namespace(namespace, sleep_duration=10)
             name, readiness, status, restarts, age = line.split
             next if status == 'Completed'
             next if status == 'Running' && /^(\d+)\/\1$/ =~ readiness
+            puts "# Waiting for: #{line}"
             ready = false
         end
         break if ready
         sleep sleep_duration
     end
 end
 
+# Show the status of a Kubernetes namespace
+def show_pods_for_namespace(namespace)
+  output = capture("kubectl get pods --namespace #{namespace} --no-headers")
+  output.each_line do |line|
+    puts "# Space: #{line}"
```

Comment:
What's with the prefix? (And why isn't letting `kubectl get pods` print its table directly good enough?)

Reply:
Prefix - Wanted to distinguish output from whatever is before and after.

Print table directly - Right, that should work as well. I guess I was on one side coming from slimming down the def from the original I derived it from, and not seeing far enough.

Reply:
Will change.

Reply:
Addressed with commit 3d4f0c814edeb834cfd312ace6db689d119889b1